docker-compose.docker.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  alchemy:
+    restart: always
+
+  graph-node:
+    restart: always
+
+  ipfs:
+    restart: always
+
+  postgres:
+    restart: always
+
```

Comment:
whats this is for ?

Reply:
This is for keeping services working. It is from [this docs](https://docs.docker.com/compose/production/#modify-your-compose-file-for-production)